### PR TITLE
Feat misc adjustments for days 1 and 2

### PR DESCRIPTION
**Pull request by yuri**
Expanding Part 3 and misc fixes to other parts for the first two days.

### CODE REVIEW

**Comment by yuri** [comment]
@SethDavenport: if you have a chance to review quickly, that would be great. If not, I'll merge myself in the afternoon.

@n1cholasv @alfredya: FYI.

File: handout/03-javascript.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+```javascript
+  var ngCourseModule  = angular.module('ngcourse');
+
+  ngCourseModule.controller('MainCtrl', function($scope){ ... });
+
+  ngCourseModule.controller('TaskListCtrl', function($scope){ ... });
+```
+
+Not that the return value of `ngCourseModule.controller()` is the same object
```
Comment (SethDavenport) [inline handout/03-javascript.md]: Not[e]

File: handout/03-javascript.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+as `ngCourseModule`.
+
+This style may seem more intuitive to those coming to JavaScript from other
+languages. There is a reason, however, why we do not use this style. The
+example above defines a new variable `ngCourseModule`. Unfortunately, a
+variable defined outside of a function becomes _global_ in JavaScript. We'll
+come back to global variables and ways to avoid them in a short while.
+
+For now, however, let's not that chained method calls provide us with a way of
```
Comment (SethDavenport) [inline handout/03-javascript.md]: not[e]

**Comment by SethDavenport** [comment]
Couple of typos, otherwise LGTM.

**Comment by yuri** [comment]
Thanks, I'll fix those with another PR tonight.

**Pull request merged**